allauth/socialaccount/providers/saml/utils.py

```diff
@@ -92,13 +99,27 @@ def build_saml_config(request, provider_config, org):
         "wantAssertionsEncrypted": avd.get("want_assertion_encrypted", False),
         "wantAssertionsSigned": avd.get("want_assertion_signed", False),
         "wantMessagesSigned": avd.get("want_message_signed", False),
-        "wantNameId": False,
+        "nameIdEncrypted": avd.get("name_id_encrypted", False),
+        "wantNameIdEncrypted": avd.get("want_name_id_encrypted", False),
+        "allowSingleLabelDomains": avd.get("allow_single_label_domains", False),
+        "rejectDeprecatedAlgorithm": avd.get("reject_deprecated_algorithm", True),
+        "wantNameId": avd.get("want_name_id", False),
+        "wantAttributeStatement": avd.get("want_attribute_statement", True),
+        "allowRepeatAttributeName": avd.get("allow_repeat_attribute_name", True),
     }
     saml_config = {
         "strict": avd.get("strict", True),
         "security": security_config,
     }
 
+    contacts = provider_config.get("contacts")
```

Comment:
This one is the only one not following the original name, how about `contact_person` ?

Reply:
I've updated the variable to match the pattern. I was torn since it's not a single person (which is confusing in the original library).  

For anyone wanting to know what that layout looks like, this is what I've found is works. Any of the contact types below can be used but shouldn't be required.
```
{
  "contactPerson": {
    "technical": {"givenName": "", "emailAddress": ""},
    "administrative": {"givenName": "", "emailAddress": ""},
    "security": {"givenName": "", "emailAddress": ""},
    "support": {"givenName": "", "emailAddress": ""},
    "billing": {"givenName": "", "emailAddress": ""},
    "other": {"givenName": "", "emailAddress": ""}
  }, 
  "organization": {
    "en-US": {
        "name": "", "displayname": "", "url": ""
    }
  }
}
```